### ISSUE TITLE
Compatibility with PHP 7

### ISSUE THREAD

**Issue by Megachip** [glpi-project/glpi#281]
looks like the glpi 0.9 isn't yet compatible with php 7 :(

```
  *** PHP Notice(8): Array to string conversion
  Backtrace :
  front/plugin.form.php:49                           

  *** PHP Notice(8): Undefined property: Plugin::$Array
  Backtrace :
  front/plugin.form.php:49  
```

**Comment by orthagh** [comment]
Could you test current master (0.90.1 will be released very soon)

Some commits fix that issue especially 00fa6771e7abe2231dbf49f0e9d112869ccc99cb

**Comment by Megachip** [comment]
thx a lot. Found this after opening this issue. Works fine on 0.9 and 0.85.5
